index.md

```diff
@@ -1,131 +1,198 @@
+have blue skies now that drastically fewer cars and planes are polluting our environment. It also showed that 
+people who chronically live with bad air are much more likely to be negatively impacted by the coronavirus.
 
-We can take the lessons learned during the pandemic and reduce our impact on our planet? How can we commute less? Can we have more online meetings and travel less to business meetings? What sustainable technologies can we invest in to displace the polluting technologies?
+We can take the lessons learned during the pandemic and reduce our impact on our planet? How can we commute less? 
+Can we have more online meetings and travel less to business meetings? What sustainable technologies can we 
+invest in to displace the polluting technologies?
 
-Now that many sectors of greenhouse gas emitting industries are idled, we’ve drastically reduced our global carbon footprint. We must work hard to keep our CO2 emissions as low as possible as we return to a post Coronavirus world. We must protect our air, our oceans, our forests and our topsoil -- we can no longer steal from mother nature. We must invest in sustainable solutions so that we can care for mother nature. It is our only choice if we wish to survive as a species.
+Now that many sectors of greenhouse gas emitting industries are idled, we’ve drastically reduced our global 
+carbon footprint. We must work hard to keep our CO2 emissions as low as possible as we return to a post 
+Coronavirus world. We must protect our air, our oceans, our forests and our topsoil -- we can no longer steal 
+from mother nature. We must invest in sustainable solutions so that we can care for mother nature. It is our 
+only choice if we wish to survive as a species.
 
 
-Invest in Healthcare and Social Services
-----------------------------------------
+Invest in healthcare, education and social services
+---------------------------------------------------
 
-Health care is a basic human right and all governments should provide healthcare to everyone, without exception. The epidemic has clearly shown which health care systems work well and which ones do not. We should examine which countries met the challenge well and copy their systems and have our nations be similarly prepared.
+Health care is a basic human right and all governments should provide healthcare to everyone, without exception. 
+The epidemic has clearly shown which health care systems work well and which ones do not. We should examine which 
+countries met the challenge well and copy their systems and have our nations be similarly prepared.
 
-Having sufficient shelter and food for every family is also a human right. Too many governments spend money on defense and lavish projects while families go hungry and without shelter. We must hold governments accountable to provide basic human services for all of their citizens. 
+A baseline education must also be free for everyone. We should teach everyone the importance of science and math,
```

Comment:
I'd drop "and math".
Reason: US math, UK maths, and mathematics belongs to science.

Reply:
In the US, math is actually often treated distinctly from science. I feel that this is more inclusive and puts more emphasis on the both, so I'll keep it.